odh-dashboard/overlays/authentication/kustomization.yaml

```diff
@@ -33,4 +33,4 @@ patchesJson6902:
 images:
 - name: oauth-proxy
   newName: registry.redhat.io/openshift4/ose-oauth-proxy
-  digest: sha256:4bef31eb993feb6f1096b51b4876c65a6fb1f4401fee97fa4f4542b6b7c9bc46
\ No newline at end of file
+  digest: sha256:ab112105ac37352a2a4916a39d6736f5db6ab4c29bad4467de8d613e80e9bb33
```

Comment:
need update for mistream csv?

Reply:
mmmmm great question, I didn't even think about it, let me check!